http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstall.scala

```diff
@@ -133,7 +158,13 @@ class CromwellRunnerAppInstall[F[_]](config: CromwellRunnerAppConfig,
         raw"sam.baseUri=${samConfig.server}",
         raw"sam.acrPullActionIdentityResourceId=${params.billingProfileId.value}"
       )
-    } yield Values(values.mkString(","))
+
+      finalList = maybeLimits match {
+        case Some(str) => values ++ List(str)
```

Comment:
you shouldn't have to wrap in a `List` here, you can just append a single element:
```scala
case Some(str) =>  values :+ str
```